Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -369,13 +369,33 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
                 var groupElement = group.render(messageView);
                 this._currentSessionOrGroup.append(groupElement);
                 this._currentSessionOrGroup = group;
-            } else
-                this._currentSessionOrGroup.addMessageView(messageView);
+            } else this._currentSessionOrGroup.addMessageView(messageView);
```

Comment:
Style: this should still be on a separate line

Reply:
Addressed in e2c3874b5b35844fbcd028d68867b71662ae988e.